src/main.rs

```diff
@@ -449,7 +449,7 @@ impl ManParser for Type1 {
 
 impl Type1 {
     // TODO Type signature
```

Comment:
Mind removing the type signature TODOs now that you've fixed these?

Reply:
fixed!